platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -23,11 +23,13 @@
 except ImportError as e:
     raise ImportError(str(e) + "- required module not found")
 
-ext_media_module = None
-try:
-    import ext_media_api as ext_media_module
-except EnvironmentError:
-    ext_media_module = None
+# Enabled when ext_media is available
+#ext_media_module = None
+#try:
+#    import ext_media_api as ext_media_module
+#except :
+#    ext_media_module = None
+#    pass
```

Comment:
Prefer to delete unused code rather than comment it out, unless there is a very good reason otherwise.

Reply:
ext_media is pending in review/ addressing comments. Once this is committed we will enable this code.

Reply:
No, as long as you intend to uncomment it soon, that's a good enough reason for me. :)